.github/workflows/push.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: Workflow - Push
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+
+jobs:
+  submodules:
+    name: Set up submodules
+    uses: ./.github/workflows/submodules.yml
+
+  dump:
+    name: Dump katana state
+    uses: ./.github/workflows/dump.yml
+    needs: submodules
+
+  artifacts:
```

Comment:
what the need for loading after dumping here? I don't understand how the `artifacts.yml` workflow can be used tbh, because it loads artifacts and do nothing then

Reply:
will remove, I thought you needed to download it to have access from outside github workflows